src/tactic/show_term.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import tactic.core
+import tactic.tauto
+
+namespace tactic
+
+/-- Run a tactic, returning the goals as they were before running the tactic. -/
+meta def capture_terms {α : Type} (t : tactic α) : tactic (list expr × α) :=
+do
+  gs ← get_goals,
+  a ← t,
+  return (gs, a)
+
+/-- Run a tactic, and then return the pretty printed original goal. -/
+meta def pp_term {α : Type} (t : tactic α) : tactic string :=
```

Comment:
I don't think this function is useful outside of `show_term`.  I'd just inline it there.

Reply:
The only reason I un-inlined it was so that the test file could run without printing any output. This tactic is so simple that perhaps it doesn't need a test file?

Reply:
Yes, I don't think it needs a test.

Reply:
Okay, removed tests, and compressed. I moved one more example into the docstring.